ChakraCore Samples/Hello World/Python/helloWorld.py

```diff
@@ -0,0 +1,48 @@
+# -------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+# -------------------------------------------------------------------------------------------------------
+
+from ctypes import *
+
+# For Windows, use ChakraCore.dll
+chakraCore = CDLL("./BuildLinux/Debug/libChakraCore.so")
+
+# You are expected to initialize the DllMain manually
+# Attach process
+chakraCore.DllMain(0, 1, 0);
```

Comment:
> You are expected to initialize the DllMain manually

Can you elaborate more here? Trying the sample on Windows with py 2.7.5 and getting ```AttributeError: function 'DllMain' not found``` on Line 13. Googling doesn't really help me unfortunately. 

Depends on what's the issue here, adding some content in readme may be helpful.